pheno_utils/config.py

```diff
@@ -100,6 +102,25 @@ def copy_tre_config():
 
 
 # %% ../nbs/00_config.ipynb 7
+def get_dictionary_properties_file_path() -> str:
+    """
+    Get the file path for dictionary properties - TODO: move to config file or DB.
+    At this point only includes field_type properties.
+
+    Args:
+
+    Returns:
+        str: the path to the file
+    """
+    path = os.path.join(DATASETS_PATH, 'metadata', '2 - Dictionary properties - field_type.csv')
```

Comment:
not my favorite file name, but I won't insist.

Reply:
I agree, but this is consistent with the original file
I hope this is temporary until we can store it in a different location, such as Athena or another type of storage service